src/Features/CSharp/Portable/CodeRefactorings/MoveDeclarationNearReference/MoveDeclarationNearReferenceCodeRefactoringProvider.Rewriter.cs

```diff
@@ -1,47 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System.Linq;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Microsoft.CodeAnalysis.Formatting;
-
-namespace Microsoft.CodeAnalysis.CSharp.CodeRefactorings.MoveDeclarationNearReference
-{
-    internal partial class MoveDeclarationNearReferenceCodeRefactoringProvider
-    {
-        private class Rewriter : CSharpSyntaxRewriter
```

Comment:
Old rewriter-based fix was removed as it's much simpler to express with the new SyntaxEditor API.